src/main/java/ConsoleOut/ConsoleOutManagement.java

```diff
@@ -0,0 +1,21 @@
+package ConsoleOut;
+import java.util.Map;
+
+public class ConsoleOutManagement {
```

Comment:
I want to create this class as an instance of writer in main class. It should implement UserWriter interface. In addition why is it called managment? Managment for me is a class to manage something. Where are methods for managing? We should call it ConsoleWriter.